guides/common/modules/proc_viewing-booted-container-images.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Viewing_booted_container_images_{context}"]
+= Viewing booted container images
+
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Prerequisites
+* *Pull-Mode Remote Execution*: If you use pull-mode remote execution, the `x`, `y`, and `z` packages must be included in the image mode host's container image.
```

Comment:
Brian, the x, y, and z were examples. You weren't supposed to use them literally.

Reply:
> Brian, the x, y, and z were examples. You weren't supposed to use them literally.

Sorry, Lena. I meant to use those as a placeholder until I get clarification on the specific packages. I should've made an additional comment after my push. I've got the question out to my team with Ian out of town.

Reply:
Added the package and will push.